src/cobalt-launcher.c

```diff
@@ -112,23 +112,36 @@ gboolean cobalt_launcher_read_flags_file(CobaltLauncher *launcher, GFile *file,
       continue;
     }
 
-    if (g_str_has_prefix(line, ENABLE_FEATURES_FLAGFILE_PREFIX) ||
-        g_str_has_prefix(line, DISABLE_FEATURES_FLAGFILE_PREFIX)) {
-      const char *feature = strchr(line, '=') + 1;
-      if (*feature == '\0') {
-        g_warning("Line in '%s' has an empty feature: %s", g_file_peek_path(file), line);
+    {
+      gint tokenc;
+      gchar **tokenv;
```

Comment:
This will leak the tokens after this finishes, you can make this `g_auto(GStrv) tokenv = NULL;` instead to "fix" that.

Reply:
Wouldn't putting it in its own scope prevent it from leaking tokens?